src/actions/auger/auger_train.ts

```diff
@@ -0,0 +1,488 @@
+import * as httpRequest from "request-promise-native"
+import * as semver from "semver"
+import * as winston from "winston"
+import * as Hub from "../../hub"
+import { FileInfo, Poller, Transaction } from "./poller"
+import { Queue } from "./queue"
+
+const AUGER_URL = "https://app.auger.ai/api/v1"
+const MAX_TOTAL_TIME_MINS = "60"
+const MAX_N_TRIALS = "100"
+const MAX_FILE_ROWS = 10000
+
+export class AugerTrainAction extends Hub.Action {
+  name = "auger"
+  label = "Auger - AutoML Training"
+  iconName = "auger/auger.png"
+  description = "Send data to Auger to start training."
+  supportedActionTypes = [Hub.ActionType.Query]
+  supportedFormattings = [Hub.ActionFormatting.Unformatted]
+  supportedVisualizationFormattings = [Hub.ActionVisualizationFormatting.Noapply]
+  requiredFields = []
+  usesStreaming = true
```

Comment:
Please add `executeInOwnProcess=true` and `extendedAction = true` below here to make sure that it executes in a different queue.